api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/MultiValueEditSubmission.java

```diff
@@ -336,6 +333,23 @@ public void addLiteralToForm(EditConfigurationVTwo editConfig, FieldVTwo field,
             log.debug("could not find value for parameter " + var  );
         }
     }
+
+    private boolean isLangSetInFirstLiteral(EditConfigurationVTwo editConfig) {
+        Map<String, List<Literal>> literalsInScope = editConfig.getLiteralsInScope();
+        if (!literalsInScope.containsKey(LABEL)) {
+            return false;
+        }
+        List<Literal> labelLiterals = literalsInScope.get(LABEL);
+        if (labelLiterals.size() == 0) {
```

Comment:
Aside: I am not a fan of all this uncertainty in code. Would be to be certain an empty list was not added to the map, would be nice to be certain a null was not added to said list, etc.

With that said, shouldn't we check to see if the list in the map is not null?